src/App.js

```diff
@@ -63,38 +104,25 @@ const HeaderWrapper = styled.div`
   top: 0;
   left: 0;
   right: 0;
-  margin: 0 auto;
   width: 100vw;
   max-width: 1240px;
   z-index: 10;
 `
 const MainWrapper = styled.main`
-  padding: 10px;
-  margin: 60px auto;
+  padding: 70px 10px 60px 10px;
   scrollbar-width: none;
   &::-webkit-scrollbar {
     display: none;
   }
 `
-const SaveGameFormWrapper = styled.div`
-  display: grid;
-  grid-gap: 20px;
-`
 const FooterWrapper = styled(Footer)`
   position: fixed;
   bottom: 0;
   left: 0;
   right: 0;
-  margin: 0 auto;
   display: flex;
   justify-content: space-evenly;
   align-items: center;
   width: 100vw;
   max-width: 1240px;
```

Comment:
Do you really need all the styling inside your app, or can you define more of it in the `Footer` and `Header` components? Isn't the width information inherited by the app wrapper?